BLE_GattServer_ExperimentalServices/README.md

```diff
@@ -0,0 +1,56 @@
+# Link Loss Service
+
+## Introduction
+This application sends an alert message over a serial terminal once the connection to the client is lost without prior warning.
+The client sets the alert message string by writing a valid byte to a R/W GATT Characteristic inside the Link Loss Service (LLS):
```

Comment:
I wouldn't mention string, it might be confusing, what to you think of replacing the `alert message string` by `alert level` ?